.doc_gen/metadata/services.yaml

```diff
@@ -50,6 +60,16 @@ cognito-identity:
   short: "&COGID;"
   sort: "Cognito Identity"
   version: cognito-identity-2014-06-30
+cognito-idp:
```

Comment:
`cognito-identity-provider`
(Smithy name)

Reply:
I'll update the entry.

Reply:
Fixed.